### ISSUE TITLE
Clarify Swag section

### ISSUE THREAD

**Issue by RichardLitt** [catalyst-network/Community#5]
Right now, the README says:

> Interested in showing your support for Catalyst Network?  Tweet us @catalystNetOrg.

How about we ask people to comment in an issue, as well? It would keep people on GitHub, and wouldn't require them to open a Twitter account. As well - what sort of swag is available? Stickers? T-shirts?

**Comment by nshCore** [comment]
Yep let's update it to opening an issue, and create a tag with swag, or something along them lines.
